orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/states/SaltStates.java

```diff
@@ -407,4 +408,11 @@ private static String parseVersion(String versionCommandOutput, Optional<String>
         }
         return versionCommandOutput;
     }
+
+    public static boolean unboundClusterConfigPresentOnAnyNodes(SaltConnector sc, Target<String> target) {
+        return measure(() -> sc.run(target, "file.file_exists", LOCAL, PingResponse.class, "/etc/unbound/conf.d/00-cluster.conf"), LOGGER,
+                "Getting information about existence of unbound config took {}ms").getResult().stream()
+                .filter(map -> MapUtils.isEmpty(map) || map.values().stream().anyMatch(Boolean::booleanValue))
+                .count() > 0;
```

Comment:
could be refactored as:
```
.anyMatch(map -> MapUtils.isEmpty(map) || map.values().stream().anyMatch(Boolean::booleanValue))
```
when this could be true: `MapUtils.isEmpty(map)`?
Would this really mean there is unbound config present?

Reply:
I cannot remember why do I put that condition :D I guess it is not possible that map is empty, maybe in case of a mock tests or sthing like that